src/Consumer/SwiftMailerConsumer.php

```diff
@@ -82,6 +82,12 @@ private function sendEmail(MessageInterface $message)
             $mail->addPart($html, 'text/html');
         }
 
+        if ($attachment = $message->getValue(['attachment', 'file'])) {
```

Comment:
Shouldn't you check for the attachment name too?

Reply:
it's checked just after, and the default `null` value is ok for the attach

Reply:
We can have an attachment without explicit fileName